File: class_6/class_6/app/src/main/java/com/alejouribesanchez/class_6/Main5Activity.java
package com.alejouribesanchez.class_6;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.text.Editable;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.LinearLayout;

public class Main5Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main5);

        LinearLayout linearLayout = (LinearLayout)findViewById(R.id.activity_main5);
        linearLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Main5Activity.this, Main6Activity.class));
            }
        });

        final EditText campo1 = (EditText) findViewById(R.id.campo_dos_palabras);
        Editable texto = campo1.getText();

        int start = 0;
        int end = 0;

        for (int i = start; i < texto.length(); i++) {

            if (texto.charAt(i) == ' ') {
                end = i;
            }
        }

        campo1.setSelection(start, end);

        Button button = (Button) findViewById(R.id.button);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                campo1.selectAll();

            }
        });

    }
}

File: class_6/class_6/app/src/main/java/com/alejouribesanchez/class_6/Main5Activity.kt
package com.alejouribesanchez.class_6

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.text.Editable
import android.view.View
import android.widget.Button
import android.widget.EditText
import android.widget.LinearLayout

class Main5Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main5)

        val linearLayout = findViewById(R.id.activity_main5) as LinearLayout
        linearLayout.setOnClickListener { startActivity(Intent(this@Main5Activity, Main6Activity::class.java)) }

        val campo1 = findViewById(R.id.campo_dos_palabras) as EditText
        val texto = campo1.text

        val start = 0
        var end = 0

        for (i in start..texto.length - 1) {

            if (texto[i] == ' ') {
                end = i
            }
        }

        campo1.setSelection(start, end)

        val button = findViewById(R.id.button) as Button
        button.setOnClickListener { campo1.selectAll() }

    }
}

